### ISSUE TITLE
Whiteboard background image not showing up after set.

### ISSUE THREAD

**Issue by oldmicah** [worldanvil/worldanvil-bug-tracker#1079]
**World Anvil Username**:
oldmicah

**World / Campaign / RPG Character / Manuscript Title**:
World  Altonne SSS. (https://www.worldanvil.com/w/altonne-sss-oldmicah)

**Feature**: 
Whiteboard

**Describe the Issue**
Images added to whiteboard's background fail to appear.

**To Reproduce**

1.  - uploaded an image using https://www.worldanvil.com/world/apollo/
2.  -  it shows up fine in other my test doc article: https://www.worldanvil.com/w/altonne-sss-oldmicah/a/test-doc-document
3.   -I created a whiteboard and set the image as the background and added a sticky.. (https://www.worldanvil.com/world/whiteboard/56124ef4-b581-4691-b3da-145a8e885989)
4. -The background does not show up in the whiteboard except as a back field with white dot grid (sticky shoes up)
5. -putting the link in an article shows no background.  (sticky shoes up)

Another user on Discord has recreated this on their account.

**Expected behavior**
Expected the back ground image to show up.

**Screenshots**
If applicable, add screenshots to help explain your problem. You can add screenshots by drag & drop into this window!

**Comment by oldmicah** [comment]
Browser: Latest OS X Safari and Firefox.  Not sure what other user on Discord used.

**Comment by Ramtrap** [comment]
I am having the same issue now, as of 05SEP22. I just joined a paid subscription for a year because after all my googling, I found your video for creating whiteboards/questboards to be the best, by far. I added my images (jpgs), created a new whiteboard, and followed instructions to change background and/or add an image (e.g. Hamburger menu --> Edit --> Change Background --> Search for image --> Click the image to add and...nothing. Black screen. I tried changing to PNG images. Nothing. I tried just adding a simple image (Green + sign at the bottom, find image and click). Nothing.
Might you have another solution/fix for this planned/in development?
